src/api-reference/request/v4.endpoints.expense-resources.markdown

```diff
@@ -1,703 +1,59 @@
-  "businessPurpose": "Trip to Lyon for company training - Modification of dates and Cost center + Custom Field",
-  "canceledPostApproval": false,
-  "closed": false,
-  "creationDate": "2018-05-25T09:17:25.000Z",
-  "custom1": {
-    "value": "Training part of IT Service"
-  },
-  "custom2": {
-    "code": "CEN1",
-    "value": "54F0CBD8833CB348BD45A6C7C621C951",
-    "href": "https://us.api.concursolutions.com/list/v1/items/54F0CBD8833CB348BD45A6C7C621C951"
-  },
-  "custom3": {
-    "code": "CEN1PRO2",
-    "value": "441D6FC50766A044ACC07FF780F1BAD9",
-    "href": "https://us.api.concursolutions.com/list/v1/items/441D6FC50766A044ACC07FF780F1BAD9"
-  },
-  "custom4": {
-    "code": "TRAINING",
-    "value": "3F54AE68BA66EF49A5984E5197202A4D",
-    "href": "https://us.api.concursolutions.com/list/v1/items/3F54AE68BA66EF49A5984E5197202A4D"
-  },
-  "endDate": "2018-07-09",
-  "endTime": "19:00",
-  "everSentBack": false,
-  "expenses": [
-    {
-      "href": "https://us.api.concursolutions.com/v4/expenses/47C8AD9E382B5143BB54DC3090577C60",
-      "id": "47C8AD9E382B5143BB54DC3090577C60",
-      "template": "https://us.api.concursolutions.com/v4/expenses/{id}"
-    }
-  ],
-  "lastModified": "2018-05-25T09:38:02.000Z",
-  "mainDestination": {
-    "countryCode": "FR",
-    "countrySubDivisionCode": "FR-69",
-    "city": "Lyon, FRANCE",
-    "name": "Lyon, FRANCE"
-  },
-  "name": "Company Training - JULY 2018",
-  "owner": {
-    "href": "https://us.api.concursolutions.com/profile/v1/users/c0d9894b-98e2-48d5-86f9-1decde90dd15",
-    "id": "c0d9894b-98e2-48d5-86f9-1decde90dd15",
-    "template": "https://us.api.concursolutions.com/profile/v1/users/{id}"
-  },
-  "pendingApproval": true,
-  "policy": {
-    "id": "F4C8BD31CA9D4D6292795BE687EB9B2A"
-  },
-  "requestId": "333X",
-  "startDate": "2018-07-07",
-  "startTime": "06:15",
-  "submitDate": "2018-05-25T09:38:02.000Z",
-  "totalApprovedAmount": {
-    "value": 123.56,
-    "currency": "USD"
-  },
-  "totalPostedAmount": {
-    "value": 123.56,
-    "currency": "USD"
-  },
-  "totalRemainingAmount": {
-    "value": 123.56,
-    "currency": "USD"
-  },
-  "travelAgency": {
-    "href": "https://us.api.concursolutions.com/v4/travelagencies/2EC038D7C3CBBE4ABA0914425064D34F",
-    "id": "2EC038D7C3CBBE4ABA0914425064D34F",
-    "template": "https://us.api.concursolutions.com/v4/travelagencies/{id}"
-  },
-  "operations": [
-    {
-      "rel": "recall",
-      "href": "https://us.api.concursolutions.com/v4/requests/053A479B3C9DD847B02A203C657AE26B/recall"
-    },
-    {
-      "rel": "cancel",
-      "href": "https://us.api.concursolutions.com/v4/requests/053A479B3C9DD847B02A203C657AE26B/cancel"
-    }
-  ]
-}
-```
+### HTTP Response
 
-## <a name="expense-resource">Expense resources</a>
+#### Payload
 
-Manage expected Expense entries attached to a Request document
+None
 
-### <a name="create-new-expected-expense-resource"></a>Create a new expected expense
+### Example
 
 #### HTTP Request
 
```

Comment:
The example response includes the URLs for datacenter/v4/expense, and datacenter/expenseconfig/v4/users, neither of which are publicly released. Expense v4 isn't even close to being released. Please ask the PM if they have a dependency on Expense Report v4, which has not begun it's CPA review.